polyply/src/gen_coords.py

```diff
@@ -32,7 +32,26 @@
 from .build_file_parser import read_build_file
 
 def find_starting_node_from_spec(topology, start_nodes):
-    start_dict = {mol_idx:None for mol_idx, _ in enumerate(topology.molecules)}
+    """
+    Given a definition of one or multiple nodes in
+    `start_nodes` find the corresponding molecule index
+    and node key in `topology`. The format of start nodes
+    is equivalent to that for annotating the residue and
+    is described in detail in the function
+    `annotate_molecules.parse_residue_spec`.
```

Comment:
```suggestion
    :meth:`annotate_molecules.parse_residue_spec`.
```
maybe method? See sphinx autodoc docs

Reply:
yes but needs full path I think